c/array-tiling/mlceu_false-unreach-call.c

```diff
@@ -0,0 +1,29 @@
+extern void __VERIFIER_error() __attribute__ ((__noreturn__));
+extern void __VERIFIER_assume(int);
+void __VERIFIER_assert(int cond) { if(!(cond)) { ERROR: __VERIFIER_error(); } }
+
+#define SIZE
```

Comment:
The same for most of the `SIZE` definitions.

Reply:
Ok. Thanks. I will upload the updated files.